pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[build-system]
```

Comment:
I think we also need to update the readme.  It makes a reference to `setup.py` in the Installation section.

Reply:
Good catch! I updated the readme.